### ISSUE TITLE
OWL import leads to NullPointerException


### ISSUE THREAD

**Issue by sternk** [spechub/Hets#1261]
Reported by **till** and assigned to **tekknix**
 Migrated from http://trac.informatik.uni-bremen.de:8080/hets/ticket/1261

***

```
Prefix: : <http://ontohub.org/monster-blend/animals/prefixTest#>
Ontology: <http://ontohub.org/monster-blend/animals/prefixTest>

Import: <https://ontohub.org/monster-blend/animals/habitats.owl>

```
leads to
```
hets: process stop! ExitSuccess
OWL parse error: null
java.lang.NullPointerException
	at org.coode.owlapi.manchesterowlsyntax.ManchesterOWLSyntaxEditorParser.parseOntology(ManchesterOWLSyntaxEditorParser.java:2057)
	at org.coode.owlapi.manchesterowlsyntax.ManchesterOWLSyntaxOntologyParser.parse(ManchesterOWLSyntaxOntologyParser.java:127)
	at uk.ac.manchester.cs.owl.owlapi.ParsableOWLOntologyFactory.loadOWLOntology(ParsableOWLOntologyFactory.java:206)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.loadOntology(OWLOntologyManagerImpl.java:829)
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.loadOntologyFromOntologyDocument(OWLOntologyManagerImpl.java:782)
	at OWL2Parser.main(OWL2Parser.java:74)

```

**Comment by cmaeder** [comment]
added https://github.com/spechub/attachment/blob/master/1261/test.omn

**Comment by cmaeder** [comment]
closed by #1479
